CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+### [Version 4.1.3](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/4.1.3) (October 7, 2022)
+
+##### Fixed
+- Fixes the value of `wzrk_c2a` key for image-only in-app notification CTAs.
+- Removes "UTM Visited" as a restricted event.
```

Comment:
@AishwaryaNanna Remove this line from the Changelog. We are not sharing publicly that UTM Visited is no longer restricted